internal/test/test.go

```diff
@@ -47,3 +49,16 @@ func CleanupConfig() {
 	config.SetOrgID("")
 	config.SetProjectID("")
 }
+
+// VerifyTemplate validates that the given template string is valid.
```

Comment:
```suggestion
// VerifyOutputTemplate validates that the given template is valid for the given value.
```

Reply:
Investigating why we did not got golangcilint issue :)

Reply:
this linter is disabled by default, the maintainers of golangcilint don't believe in it's value